packages/react-badge/src/tmp-icons.stories.tsx

```diff
@@ -0,0 +1,62 @@
+//
```

Comment:
nit: Should this have just a `.tsx` extension instead of a `.stories.tsx` one?

Reply:
>Should this have just a .tsx extension instead of a .stories.tsx one?
no.

**Why?:** 
- we wanna be explicit about content for storybook. 
- in upcoming stages of DX improvements, we'll use separate tsconfig for storybook related things -> this will allow us  to not ship those files (storybook related) to npm registry (which we currently do)